Source/SwiftLintBuiltInRules/Rules/Performance/EmptyCountRule.swift

```diff
@@ -1,7 +1,9 @@
+import SwiftLintCore
+import SwiftOperators
```

Comment:
Import not needed anymore.

Reply:
Fixed at 9b129ef

Reply:
aedd175 adds back `import SwiftLintCore` since the code won't highlight properly on my machine (but I belive it does build fine without it so could be removed if that's preferred)

Reply:
I also notice the highlighting issue and therefore prefer the import.